lib/components/input-group.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div :class="classObject">
+        <slot name="left"></slot>
```

Comment:
Input groups can have multiple left and right addons under bootstrap V4 though, and this would limit it to only a single left and right addon.

https://v4-alpha.getbootstrap.com/components/input-group/